README.md

```diff
@@ -1,19 +1,41 @@
-# React render() Must Not Have Side Effects
+# React render() must not have side effects
 
-## Objectives
+???
```

Comment:
You are questioning whether we need objectives here I think, right? Well I don't know myself. Would seem not. @AnnJohn what do you think?